src/components/validationStatus/ValidationStatus.vue

```diff
@@ -1,83 +1,83 @@
 <template>
-  <b-card-footer class="p-0 border-0">
-    <div class="statusbar d-flex align-items-center justify-content-end pr-3 border-top">
-      <div>
-        <div class="status-bar-container d-flex align-items-center justify-content-end">
-          <template v-if="autoValidate">
-            <button v-if="numberOfProblemsToDisplay === 0" type="button" class="btn btn-light" :disabled="true">
-              {{ $t('BPMN Valid') }}
-              <span class="badge badge-success badge-pill">
-                <font-awesome-icon :icon="faCheck" />
-              </span>
-            </button>
-            <button v-else type="button" data-test="validation-list-toggle" class="btn btn-light"
-              @click="shouldDisplayProblemsPanel = !shouldDisplayProblemsPanel"
-            >
-              {{ $t('BPMN Issues') }}
-              <span class="badge badge-primary badge-pill">
-                {{ numberOfProblemsToDisplay }}
-              </span>
-              <font-awesome-icon class="ml-3" :icon="chevronIcon" />
-            </button>
-          </template>
+  <b-card-footer class="p-0 border-0 statusbar d-flex align-items-center justify-content-end pr-3 pl-3 border-top">
+    <b-btn
+      class="mr-auto"
+      :disabled="!xmlManager"
+      variant="secondary"
+      size="sm"
+      data-test="downloadXMLBtn"
+      @click="xmlManager && xmlManager.download()"
+    >
+      <i class="fas fa-download mr-1"/>
+      {{ $t('Download BPMN') }}
+    </b-btn>
```

Comment:
Most of the changes in this template are whitespace changes (I removed a couple of `div`s that increased nesting but provided no real value). If you're reading this on GitHub, you can hide the whitespace changes to make it easier to review:

<img width="256" alt="Screen Shot 2020-02-19 at 1 16 11 PM" src="https://user-images.githubusercontent.com/7561061/74862457-31eb9800-531a-11ea-9f44-602633546c1e.png">

I've commented on the important bit that's changed: adding the new download button.